# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libmnl/hotfix-excludedocs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libmnl-1.0.5/doxygen.cfg.in.vanilla	2025-04-03 22:07:51.874830362 +0200
+++ libmnl-1.0.5/doxygen.cfg.in	2025-04-03 22:07:57.495859728 +0200
@@ -9,7 +9,7 @@
 INPUT                  = .
 FILE_PATTERNS          = */src/*.c
 RECURSIVE              = YES
-EXCLUDE_SYMBOLS        = EXPORT_SYMBOL mnl_nlmsg_batch mnl_socket
+EXCLUDE_SYMBOLS        = EXPORT_SYMBOL mnl_nlmsg_batch mnl_socket LIST_ENTRY
 EXAMPLE_PATTERNS       =
 INPUT_FILTER           = "sed 's/EXPORT_SYMBOL//g'"
 SOURCE_BROWSER         = YES
